src/Build.OM.UnitTests/Definition/Project_Tests.cs

```diff
@@ -4035,6 +4035,50 @@ public void ProjectImportedEventEmptyFile()
         }
 
         [Fact]
+        public void ProjectImportedEventFaultedFile()
+        {
+            using (var env = TestEnvironment.Create(_output))
+            {
+                env.SetEnvironmentVariable("MSBUILDLOGIMPORTS", "1");
+
+                const string contents = @"<?xml version=""1.0"" encoding=""utf-8""?>
+<Project>BROKEN</Project>
+";
+                string importPath = ObjectModelHelpers.CreateFileInTempProjectDirectory(Guid.NewGuid().ToString("N"), contents, Encoding.UTF8);
```

Comment:
You can use Env.CreateFile as well here, to ensure it gets deleted afterwards. TestEnvironment is new, so many older tests are not as clean as they could be.

Reply:
I pretty much copy pasted the `IgnoreEmptyImports` test. Do you want me to update that one as well?

Reply:
Sure, thanks!

Reply:
I updated both tests to use `env.CreateFile`